templates/federated_reporting/import_file.sh

```diff
@@ -60,8 +63,12 @@ COMMIT;
 EOF
 } | "$CFE_BIN_DIR"/psql -U $CFE_FR_DB_USER -d cfdb 2>&1 | "$CFE_FR_COMPRESSOR" $CFE_FR_COMPRESSOR_ARGS >"$file.log.$CFE_FR_COMPRESSOR_EXT" && {
   rm -f "$file.importing"
-  exit 0
+  ret=0
 } || {
   mv "$file.importing" "$file.failed"
-  exit 1
+  ret=1
 }
+end_time=$(date +%s)
+duration=$((end_time - begin_time))
+log "Import for $hostkey took $duration seconds"
```

Comment:
This log could be determined somewhat from timestamps in agent log, but this gives specific information about a single hostkey and more deterministic than relying on agent logs having timestamps

```
Q: ".../bin/bash /opt/": [Tue May 24 23:18:30 CEST 2022] Import for SHA=e51c4479c586bd253df1d8665d49d71338cf00001cdc8bda5d2fd115e01c028f took 8 seconds
```